ROMFS/px4fmu_common/init.d/rcS

```diff
@@ -505,6 +505,9 @@ then
 			set AUX_MODE none
 		fi
 		camera_trigger start
+
+		param set CAM_FBACK_MODE 1
```

Comment:
I'm only setting it here to force the feedback module into trigger feedback mode when camera trigger is running. Otherwise the user would have to choose it.

This will be removed anyway when we add hotshoe feedback.